src/audio/Audio.js

```diff
@@ -56,7 +56,11 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 		this.buffer = audioBuffer;
 		this.sourceType = 'buffer';
 
-		if ( this.autoplay ) this.play();
```

Comment:
Ok i will take care about that ! An other things that can cause trouble is spacement for array of vertices/indexes/matrix. Are you agree if i add some "disable comment" for eslint rules in this cases ?